src/test/java/io/pravega/connectors/presto/decoder/TestJsonRowDecoder.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (c) Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.pravega.connectors.presto.decoder;
+
+import com.facebook.presto.common.type.Type;
+import com.facebook.presto.decoder.DecoderColumnHandle;
+import com.facebook.presto.spi.ColumnHandle;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.pravega.connectors.presto.PravegaColumnHandle;
+import io.pravega.connectors.presto.PravegaRecordValue;
+import io.pravega.connectors.presto.TypedRecordValue;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static com.facebook.presto.common.type.BigintType.BIGINT;
+import static com.facebook.presto.common.type.VarcharType.VARCHAR;
+import static com.google.common.collect.ImmutableSet.toImmutableSet;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+@Test
+public class TestJsonRowDecoder
```

Comment:
Minor: Let's follow the pattern of adding test to the end of the class name,  `JsonRowDecoderTest` instead in this case.

Reply:
renamed